hw1-app/src/app/courses/models/course-item.model.ts

```diff
@@ -12,13 +13,15 @@ export class Course implements ICourse {
   public creationDate: Date;
   public duration: number;
   public description: string;
+  public topRated: boolean;
 
   constructor(model?: ICourse) {
```

Comment:
you can improve  constructor(model: Partial<ICourse> = {}) {  this.id = model.id || '';  this.title = model.title || '';

Reply:
minor cooment